src/main/java/org/jenkinsci/plugins/workflow/support/steps/build/BuildTriggerListener.java

```diff
@@ -29,6 +30,11 @@ public void onStarted(Run<?, ?> run, TaskListener listener) {
                     TaskListener taskListener = stepContext.get(TaskListener.class);
                     // encodeTo(Run) calls getDisplayName, which does not include the project name.
                     taskListener.getLogger().println("Starting building: " + ModelHyperlinkNote.encodeTo("/" + run.getUrl(), run.getFullDisplayName()));
+                    FlowNode parentNode = stepContext.get(FlowNode.class);
+                    if (parentNode != null) {
```

Comment:
No need to check for null if defined via `@StepContextParameter` (which is actually deprecated but fixing that requires a larger change).